scripts/release.sh

```diff
@@ -74,13 +74,13 @@ case "$response" in
 esac
 
 # generate the changelog for the git release update
-releaseChangelog=$(git log --pretty=format:"- [%as] %s (%h)" $(git describe --tags --abbrev=0 @^)..@ --abbrev=7 | sed '/[🔧🎬⬆️📸✅💡📝]/d')
+releaseChangelog=$(git log --pretty=format:"- [%as] %s (%h)" $(git describe --tags --abbrev=0 @^)..@ --abbrev=7 | sed '/[🔧🎬⬆️📸✅💡📝🔖]/d')
 releaseTempFile=$(mktemp)
 echo "$releaseChangelog" >> $releaseTempFile
 
 # generate the updated CHANGELOG.md for the repo (Flutter requirement)
 changelog="## $newVersion\n"
-changelog+=$(git log --pretty=format:"* %s (%h)" $(git describe --tags --abbrev=0 @^)..@ --abbrev=7 | sed '/[🔧🎬⬆️📸✅💡📝]/d')
+changelog+=$(git log --pretty=format:"* %s (%h)" $(git describe --tags --abbrev=0 @^)..@ --abbrev=7 | sed '/[🔖]/d')
```

Comment:
We want this changelog to include all commit?

Reply:
I'm thinking so, as I noticed that otherwise in cases like this it would just be an empty section? This is for the CHANGELOG.md in the repo that Flutter requires, which I think maybe makes sense to be more comprehensive than the github release log? What do you think?